src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -91,6 +92,13 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchCurrentUser(currentUser);
     }
 
+    componentDidUpdate({ refreshIdentity: prevRefreshIdentity }: Props) {
+        const { refreshIdentity } = this.props;
+        if (refreshIdentity !== prevRefreshIdentity) {
```

Comment:
Can we expose a similar `refresh` method from here that the parent can call?

Reply:
Sounds reasonable, let me work on that